### PR TITLE
Fixing sourcify article images url

### PR DESCRIPTION
**Pull request by edisinovcic**

### CODE REVIEW

**Comment by edisinovcic** [comment]
@franzihei

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Think about the following: How many developers of the dapps you are using do you
 
 Let’s imagine you want to interact with a dapp using MetaMask and it shows you this dialogue: 
 
-![Yolo Signing](../img/2020/05/MetaMask_Yolosign.png)
+![Yolo Signing](https://raw.githubusercontent.com/ethereum/solidity-blog/67a114759bcec663a094814e813e32d2e7d164f1/img/2020/05/MetaMask_Yolosign.png)
```
Comment (axic) [inline _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:26]: Still the easiest (not depending on github) is: `/img/2020/...` as mentioned 2 PRs ago :)

Comment (franzihei) [reply _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:26]: but it didn't work? wasn't that what I tried just before?

Comment (axic) [reply _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:26]: You guys tried `./` and `../`. I'm saying `/`.

Comment (franzihei) [reply _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:26]: I didn't try `./` but yes doh, I missed the `/` here https://github.com/ethereum/solidity-blog/pull/60. Didn't see your comment, sorry! I prefer only the `/`. Let's change it into that.

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ Sourcify is a project on the Ethereum blockchain that:
 * Checks that it is the same as the deployed code.
 * Permanently stores the source code, ABI and other metadata on IPFS.
 
-![Sourcify Flowchart](../img/2020/05/Sourcify_Flowchart.png)
+![Sourcify Flowchart](https://raw.githubusercontent.com/ethereum/solidity-blog/67a114759bcec663a094814e813e32d2e7d164f1/img/2020/05/Sourcify_Flowchart.png)
```
Comment (franzihei) [inline _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:69]:
```suggestion
![Sourcify Flowchart](/img/2020/05/Sourcify_Flowchart.png)
```

File: _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md (hunk 1 above)
Comment (franzihei) [inline _posts/2020-05-29-Sourcify-Towards-Safer-Contract-Interaction-for-Humans.md:26]:
```suggestion
![Yolo Signing](/img/2020/05/MetaMask_Yolosign.png)
```

**Comment by chriseth** [comment]
Please squash :)

**Pull request merged**